app.go

```diff
@@ -173,16 +197,11 @@ func (app *App) Run() {
 	}
 }
 
-// Start starts the application.
+// Start executes all the onStart hooks of the fx.Lifecycle.
```

Comment:
Yeah. I was trying to cut some corners for the sake of opening this PR, totally agree on making this godoc better.